### PR TITLE
Change adapter to @adapts decorator.

### PR DESCRIPTION
**Pull request by thet**

### CODE REVIEW

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

File: news/3876.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Change adapter to @adapts decorator.
```
Comment (gforcada) [inline news/3876.bugfix:1]: Like on the other PR, `adapts to @adapter` 😄

Comment (thet) [reply news/3876.bugfix:1]: brain's batch processing errors :D

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — APPROVED** [review]

**Pull request merged**